Added some fuctions from 1.9.0

OH1KH opened the pull request:
Now when wsjt-x has become very popular the CQ-monitor is more than needed.
Also seems that I got fixed QRZ data disappearing at wsjt-x logging phase.

Some other additions, too. Because of own needs + requests from local hams.

What I forgot to write to README is that you need to install and compile lpk under richmemo folder.
For me it made one additional folder under richmemo called lib/x86_64-linux but it does not transfer with "publish". I'm not very sure how this lpk should be added properly.

ok2cqr commented:
This pull request is really big. I'd like to release version 2.0.5  before trying to merge this. Unfortunately I'm not digital ham and don't use WSJT at all. I'm afrait I won't be able to take care about the code in the future.

OH1KH commented:
12.03.2017, 21:35, Petr Hlozek kirjoitti:
>
> This pull request is really big. I'd like to release version 2.0.5 
> before trying to merge this. Unfortunately I'm not digital ham and 
> don't use WSJT at all. I'm afrait I won't be able to take care about 
> the code in the future.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/ok2cqr/cqrlog/pull/42#issuecomment-285969416>, or 
> mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AL85xBAp5aNyN9CYN0fXlXx8wpkmK0kRks5rlEkFgaJpZM4MafZS>.
>
Same things as I requested before. Just dropped from 1.9.0 to this version

At least:
Take the wsjt-x decoding part. It has buxfix. Drop away the protocol 
description I again added when studied QRZ data
dropping off while wsjt-x logging.

I suppose there is new way to request QRZ ? Another thread?
Anyway it returns so slow that logging is closed before information 
comes from QRZ.
Someone said at forum that sometimes info is set to log. It may happen 
when having very fast internet and fast PC.

With changes I made it works now and should also keep any extra own text 
added to newqso form when wsjt-x logging is closed.


WSJT-x activity has blown in hands. Activity has grown over many hundred 
percent during last year.

Wsjt-x logging is good, but CQ-monitor is MUST, with big letters, if you 
are serious wsjtx worker. Otherwise it is very hard to
clear out all worked DXCCs and Main grids on every band while monitoring 
them.

Without CQ-monitor people have to use third program (JT alert or 
similars), even under wine, to get same properties.

I know you do not work JT, but many of your program users do. What about 
them ?

Keeping up the code is problem, but when it has proven to work, do not 
change anything :) related to this.
In future there will be more digital modes. I'm sure.


Propagation_dk0wcy you can easily drop out, if you see it so. It is not 
related to anything else. But it is better here in high
latitudes than Boulder info because of different A and K index.

Locator map has some relatives to CQ-monitor, but I did not check what. 
You can easily clear it out by dropping the form
and related pascal code and see where it complains. I suppose it is just 
a small thing.

Because you are CW ham I suppose you can keep PgUP/PgDN buttons at CW keys?
If you use memories with mouse you do not have to change focus to newQso 
to get keyboard PgUP/PgDN to work.


I'm sad to notice that I will be one version late again. Again same work 
with 2.0.5. Thumbs down!


-- 
Saku
OH1KH

OH1KH commented:
I did add today two files that git transfer left out from some reason (?????) fwkd.lrs and richmemopackage.lrs
Now the repo is tested to compile with fresh computer.

OH1KH commented:
There is a bug in fMain.pas (in original)  "if frmNewQSO.mnuRemoteMode.Checked then" allowing qso edit while in wsjt-x remote. That will cause execption with focusing nonexisiting window. 
Fixed.

OH1KH commented:
Restoring newQSO/Mode:AUTO selection when coming back from wsjt-x remote was somehow lost some day. All other code was there, but restoring at Disbleremotemode was missing. Added.

ok2cqr commented:
it sould work even without fwkd.lrs and richmemopackage.lrs. They are generated by Lazarus itself and some time ago, I added *.lrs to ignore list.

OH1KH commented:
15.03.2017, 15:27, Petr Hlozek kirjoitti:
>
> it sould work even without fwkd.lrs and richmemopackage.lrs. They are 
> generated by Lazarus itself and some time ago, I added *.lrs to ignore 
> list.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub 
> <https://github.com/ok2cqr/cqrlog/pull/42#issuecomment-286740481>, or 
> mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AL85xBKvAIWOUkHJ6OJMY9FkGU04mre0ks5rl-dKgaJpZM4MafZS>.
>
No.

I do not know what is inside richmemopackage.lrs.

fwkd.lrs contains map image used in worked locators.  Notice name is bit 
different that unit name.

-- 

Saku
